backend/app/views/spree/admin/adjustment_reasons/index.html.erb

```diff
@@ -33,7 +33,7 @@
             <%= adjustment_reason.name %>
           </td>
           <td class="align-center">
-            <%= Spree.t(adjustment_reason.active? ? :active : :inactive) %>
+            <%= Spree::AdjustmentReason.human_attribute_name(adjustment_reason.active? ? :active : :inactive) %>
```

Comment:
I don't think this is an attribute name, it's translations for two of the values the attribute can have.